DirectProgramming/C++SYCL_FPGA/Tutorials/DesignPatterns/onchip_memory_cache/README.md

```diff
@@ -1,29 +1,43 @@
-# Caching On-Chip Memory to Improve Loop Performance
+# `On-Chip Memory Cache` Sample
+
 This FPGA tutorial demonstrates how to build a simple cache (implemented in FPGA registers) to store recently-accessed memory locations so that the compiler can achieve II=1 on critical loops in task kernels.
 
+| Area                 | Description
+|:--                   |:--
+| What you will learn  | How and when to implement the on-chip memory cache optimization
+| Time to complete     | 30 minutes
+| Category             | Code Optimization
+
+## Purpose
+
+This sample demonstrates the following concepts:
+
+- How to implement the on-chip memory cache optimization technique
+- The scenarios in which this technique benefits performance
+- How to tune the cache depth
+
+## Prerequisites
+
 | Optimized for                     | Description
 ---                                 |---
-| OS                                | Linux* Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
-| Hardware                          | Intel® Agilex™, Arria® 10, and Stratix® 10 FPGAs
+| OS                                | Ubuntu* 18.04/20.04 <br> RHEL*/CentOS* 8 <br> SUSE* 15 <br> Windows* 10
+| Hardware                          | Intel Agilex® FPGA <br> Intel® Arria® 10 FPGA <br> Intel® Stratix® 10 FPGA
 | Software                          | Intel® oneAPI DPC++/C++ Compiler
-| What you will learn               | How and when to implement the on-chip memory cache optimization
-| Time to complete                  | 30 minutes
 
 > **Note**: Even though the Intel DPC++/C++ OneAPI compiler is enough to compile for emulation, generating reports and generating RTL, there are extra software requirements for the simulation flow and FPGA compiles.
 >
-> For using the simulator flow, Intel® Quartus® Prime Pro Edition and one of the following simulators must be installed and accessible through your PATH:
+> For using the simulator flow, you must have Intel® Quartus® Prime Pro Edition and one of the following simulators installed and accessible through your PATH:
 > - Questa*-Intel® FPGA Edition
 > - Questa*-Intel® FPGA Starter Edition
-> - ModelSim® SE
+> - ModelSim SE
 >
 > When using the hardware compile flow, Intel® Quartus® Prime Pro Edition must be installed and accessible through your PATH.
->
-> :warning: Make sure you add the device files associated with the FPGA that you are targeting to your Intel® Quartus® Prime installation.
 
-## Prerequisites
 
-This sample is part of the FPGA code samples.
-It is categorized as a Tier 3 sample that demonstrates a design pattern.
+> **Warning** Make sure you add the device files associated with the FPGA that you are targeting to your Intel® Quartus® Prime installation.
+
```

Comment:
Why remove the Prerequisites section?
This section is present in all the other FPGA samples, I can remove it from all of them if you think that this is better this way

Reply:
I did not remove it. I moved it further up the readme.

Reply:
Oh right, my bad